Exporting a project to iOS no longer works (regression from 4.0.alpha12 -> alpha13)

### Godot version

4.0.alpha13

### System information

macOS Monterey 12.5, MacBook Pro 16" (intel), Forward Mobile

### Issue description

When trying to export a project for iOS I get a dialog saying the export failed and the following error message `Cannot open directory '{my specific path}/Images.xcassets/LaunchImage.launchimage/'.`

The export worked in alpha 12.

full log:
```
[ERROR]  Transient parent has another exclusive child.
Static framework: libgodot.ios.debug.xcframework
Unzipping...
READ: data.pck
READ: godot_ios/
READ: godot_ios/Images.xcassets/
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Contents.json
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Contents.json size: 2567
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-480h@2x.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-480h@2x.png size: 347
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-568h@2x.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-568h@2x.png size: 395
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-667h@2x.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-667h@2x.png size: 514
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-Landscape-736h@3x.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-Landscape-736h@3x.png size: 1506
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-Landscape-X.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-Landscape-X.png size: 1403
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-Landscape.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-Landscape.png size: 466
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-Landscape@2x.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-Landscape@2x.png size: 1746
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-Portrait-736h@3x.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-Portrait-736h@3x.png size: 1429
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-Portrait-X.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-Portrait-X.png size: 1426
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-Portrait.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-Portrait.png size: 421
READ: godot_ios/Images.xcassets/LaunchImage.launchimage/Default-Portrait@2x.png
ADDING: godot_Images.xcassets/LaunchImage.launchimage/Default-Portrait@2x.png size: 1615
READ: godot_ios/Images.xcassets/SplashImage.imageset/
READ: godot_ios/Images.xcassets/SplashImage.imageset/Contents.json
ADDING: godot_Images.xcassets/SplashImage.imageset/Contents.json size: 343
READ: godot_ios/Images.xcassets/SplashImage.imageset/splash@2x.png
ADDING: godot_Images.xcassets/SplashImage.imageset/splash@2x.png size: 14779
READ: godot_ios/Images.xcassets/SplashImage.imageset/splash@3x.png
ADDING: godot_Images.xcassets/SplashImage.imageset/splash@3x.png size: 14779
READ: godot_ios/en.lproj/
READ: godot_ios/en.lproj/InfoPlist.strings
ADDING: godot_en.lproj/InfoPlist.strings size: 45
READ: godot_ios/vulkan/
READ: godot_ios/vulkan/icd.d/
READ: godot_ios/vulkan/icd.d/MoltenVK_icd.json
ADDING: godot_vulkan/icd.d/MoltenVK_icd.json size: 166
READ: godot_ios/Launch Screen.storyboard
ADDING: godot_Launch Screen.storyboard size: 2991
READ: godot_ios/dummy.cpp
ADDING: godot_dummy.cpp size: 2215
READ: godot_ios/dylibs/
READ: godot_ios/dylibs/empty
ADDING: godot_dylibs/empty size: 42
READ: godot_ios/export_options.plist
ADDING: godot_export_options.plist size: 493
READ: godot_ios/godot_ios-Info.plist
ADDING: godot_ios-export-Info.plist size: 1745
READ: godot_ios/godot_ios.entitlements
ADDING: godot_ios-export.entitlements size: 221
READ: godot_ios.xcodeproj/
READ: godot_ios.xcodeproj/project.xcworkspace/
READ: godot_ios.xcodeproj/project.xcworkspace/contents.xcworkspacedata
ADDING: ios-export.xcodeproj/project.xcworkspace/contents.xcworkspacedata size: 156
Creating /Users/davidmagnusson/Developer/Developer/export/ios-export.xcodeproj/project.xcworkspace
READ: godot_ios.xcodeproj/xcshareddata/
READ: godot_ios.xcodeproj/xcshareddata/xcschemes/
READ: godot_ios.xcodeproj/xcshareddata/xcschemes/godot_ios.xcscheme
ADDING: ios-export.xcodeproj/xcshareddata/xcschemes/ios-export.xcscheme size: 3458
Creating {my specific path}/export/ios-export.xcodeproj/xcshareddata/xcschemes
READ: godot_ios.xcodeproj/project.pbxproj
ADDING: ios-export.xcodeproj/project.pbxproj size: 14377
READ: libgodot.ios.debug.xcframework/
READ: libgodot.ios.debug.xcframework/ios-arm64_x86_64-simulator/
READ: libgodot.ios.debug.xcframework/ios-arm64_x86_64-simulator/empty
READ: libgodot.ios.debug.xcframework/ios-arm64_x86_64-simulator/libgodot.a
ADDING: ios-export.xcframework/ios-arm64_x86_64-simulator/libgodot.a size: 119544368
READ: libgodot.ios.debug.xcframework/Info.plist
ADDING: ios-export.xcframework/Info.plist size: 1028
READ: libgodot.ios.debug.xcframework/ios-arm64/
READ: libgodot.ios.debug.xcframework/ios-arm64/empty
READ: libgodot.ios.debug.xcframework/ios-arm64/libgodot.a
ADDING: ios-export.xcframework/ios-arm64/libgodot.a size: 117187120
READ: libgodot.ios.release.xcframework/
READ: libgodot.ios.release.xcframework/Info.plist
READ: libgodot.ios.release.xcframework/ios-arm64/
READ: libgodot.ios.release.xcframework/ios-arm64/empty
READ: libgodot.ios.release.xcframework/ios-arm64/libgodot.a
READ: libgodot.ios.release.xcframework/ios-arm64_x86_64-simulator/
READ: libgodot.ios.release.xcframework/ios-arm64_x86_64-simulator/empty
READ: libgodot.ios.release.xcframework/ios-arm64_x86_64-simulator/libgodot.a
READ: MoltenVK.xcframework/
READ: MoltenVK.xcframework/ios-arm64_x86_64-simulator/
READ: MoltenVK.xcframework/ios-arm64_x86_64-simulator/libMoltenVK.a
ADDING: MoltenVK.xcframework/ios-arm64_x86_64-simulator/libMoltenVK.a size: 18639328
READ: MoltenVK.xcframework/ios-arm64/
READ: MoltenVK.xcframework/ios-arm64/libMoltenVK.a
ADDING: MoltenVK.xcframework/ios-arm64/libMoltenVK.a size: 29340592
READ: MoltenVK.xcframework/Info.plist
ADDING: MoltenVK.xcframework/Info.plist size: 2078
Using Launch Images
[ERROR]  Cannot open directory '{my specific path}/export/ios-export/Images.xcassets/LaunchImage.launchimage/'.

```

### Steps to reproduce

Just try to export a project for iOS, a fresh project worked for reproducing.

### Minimal reproduction project

_No response_

--- a/platform/ios/export/export_plugin.cpp
+++ b/platform/ios/export/export_plugin.cpp
@@ -1528,8 +1528,6 @@ Error EditorExportPlatformIOS::export_project(const Ref<EditorExportPreset> &p_p
 
 		//write
 
-		file = file.replace_first("ios/", "");
-
 		if (files_to_parse.has(file)) {
 			_fix_config_file(p_preset, data, config_data, p_debug);
 		} else if (file.begins_with("libgodot.ios")) {

